src/fontra/client/web-components/reference-font.js

```diff
@@ -49,7 +49,7 @@ export class ReferenceFont extends UnlitElement {
     return this.controller.model;
   }
 
-  render() {
+  async render() {
```

Comment:
I think the render() method should stay sync (it should run fast). It can still trigger loading of the fonts, but it can add them to the list UI when it's done. So, either you should use a .then() on the loading result, _or_ add another method that is async to load the fonts, but call it _without_ awaiting. I prefer the latter solution, as it also nicely factors out font loading from the opfs.